react-native/Fable.Helpers.ReactNative.fs

```diff
@@ -108,11 +129,320 @@ module Props =
     | [<CompiledName("locked-closed")>] LockedClosed
     | [<CompiledName("locked-open")>] LockedOpen
 
+    [<StringEnum; RequireQualifiedAccess>]
+    type AlignSelf =
+        | Auto 
+
+    [<StringEnum; RequireQualifiedAccess>]
+    type TextDecorationStyle = 
+        | Solid | Double | Dotted | Dashed
+
+    [<StringEnum; RequireQualifiedAccess>]
+    type FontStyle = 
+        | Normal | Italic 
+        
+    [<StringEnum; RequireQualifiedAccess>]
+    type WritingDirection =
+        | Auto | Ltr | Rtl 
```

Comment:
Personally, I'd want to stick to the original RN names because searching for usage help might be easier that way. In this specific case `Rtl` and `Ltr` are common UI design abbreviations.